xbmc/video/dialogs/GUIDialogCMSSettings.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+ *      Copyright (C) 2005-2014 Team XBMC
```

Comment:
@Memphiz what's the right way to do this? I'm hesitant to touch the copyright notices for others :)